### PR TITLE
TAUR-1272 Update rabbitmq dependency information

### PR DESCRIPTION
**Pull request by unixorn**
* Stop attempting to install rabbitmq_management if it is already installed.
* Ensure that rabbitmq-server is restarted after installing the management plugin

### CODE REVIEW

**Comment by unixorn** [comment]
@jcasner @shantanoo @oxtopus please CR.

File: infrastructure/saltcellar/rabbitmq/init.sls [hunk 1]
```diff
@@ -43,12 +43,21 @@ rabbitmq-server:
...
 
 enable-rabbitmq-management:
   cmd.run:
     - name: /usr/lib/rabbitmq/bin/rabbitmq-plugins enable rabbitmq_management
     - require:
-      - pkg: rabbitmq-server
       - service: rabbitmq-server
+    - unless: grep rabbitmq_management /etc/rabbitmq/enabled_plugins
+    - watch_in:
+      - cmd: kick-rabbit-service
```
Comment (jcasner) [inline infrastructure/saltcellar/rabbitmq/init.sls]: s/`kick`/`restart` throughout

**Comment by unixorn** [comment]
ready for another review @jcasner

**Comment by oxtopus** [comment]
:+1:

**Pull request merged**